src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/TypeVisitor.cs

```diff
@@ -144,6 +145,7 @@ protected string Visit(IAcceptor acceptor, string name, string title, string dat
             if (attributes.Any())
             {
                 Attribute attr = attributes.OfType<OpenApiPropertyAttribute>().SingleOrDefault();
+                schema.ApplyValidationAttributes(attributes.OfType<ValidationAttribute>());
```

Comment:
I like this `ApplyValidationAttributes` method! I might need to apply this approach to other `OpenApiXXXAttribute` classes as well.

In the mean time, would you please swap both lines 147 and 148 so that your new method doesn't interrupt readability?

Reply:
Alternatively, your method can be put at the bottom of this `if(attributes.Any())` block - line 164 or so for consistency